.eslintrc.js

```diff
@@ -0,0 +1,52 @@
+module.exports = {
+	"env": {
+		"browser": true,
+		"es2021": true,
+		"node": true
+	},
+	"extends": "eslint:recommended",
+	"overrides": [
+	],
+	"parserOptions": {
+		"ecmaVersion": "latest",
+		"sourceType": "module"
+	},
+	"rules": {
+		"indent": [
```

Comment:
Since this is JS and not JSON, we should have comments explaining what each of these rules does. Or is this file generated somehow?

Reply:
Yes, it's generated with `npm init @eslint/config`. But I have added some comments. Please feel free to remove any you find not needed.

Reply:
Are we ever expected to regenerate this, or is this just for the initial file?

Reply:
We don't expect it ever to be re-generated. We just needed something to start with.